compiler/bytecode/expression_generation_test.go

```diff
@@ -71,6 +71,62 @@ func TestIfExpressionWithoutAlternativeCompilation(t *testing.T) {
 	compareBytecode(t, bytecode, expected)
 }
 
+func TestIfExpressionWithElsifCompilation(t *testing.T) {
```

Comment:
I can vaguely understand what the  vm codes are trying to do, still I"m not that confident with it. So, I would really appreciate it if you review here a bit more carefully than the other parts I wrote...